client/src/components/CatalogTourCard/CatalogTourCard.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Stack, CardActions, CardMedia, Button, Typography, CardContent } from '@mui/material';
+import AccessTimeIcon from '@mui/icons-material/AccessTime';
+import { ReactComponent as CoinsIcon } from '../../svg/CoinsIcon.svg';
+import styles from './CatalogTourCard.module.scss';
+
+function CatalogTourCard() {
+  return (
+    <Stack
+      direction={{ xs: 'column', tablet: 'row' }}
+      sx={{ backgroundColor: 'white', borderRadius: '10px', boxShadow: '0px 0px 40px rgba(0, 0, 0, 0.05)' }}
```

Comment:
white color could be used from our theme.

Reply:
I can't find this color in the theme. Does this mean I have to add it myself to the theme?